app/src/lib/stores/app-store.ts

```diff
@@ -3532,6 +3537,12 @@ export class AppStore extends TypedBaseStore<IAppState> {
     this.emitUpdate()
   }
 
+  public _setShowCommitLengthWarning(showCommitLengthWarning: boolean) {
+      setBoolean("showCommitLengthWarning", showCommitLengthWarning)
```

Comment:
Just noting again for other comment -> prefer the use of a constant set at the top of file.